src/test/java/io/stargate/sgv2/jsonapi/api/v1/tables/CreateTableIntegrationTest.java

```diff
@@ -322,10 +322,10 @@ private static Stream<Arguments> allTableData() {
                           """,
                   "invalidPartitionSortOrderingValueTable",
                   true,
-                  ErrorCodeV1.INVALID_REQUEST_NOT_JSON.name(),
+                  ErrorCodeV1.INVALID_REQUEST_BAD_SYNTAX.name(),
```

Comment:
Gee, figuring out how to fix this test was not easy: not a big fan of huge matrix tests because IDE makes it difficult to find out where the actual test data originates in. :-(